WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,3 +1,7 @@
-This release is short and sweet: after deleting a site, you might’ve noticed that same site sticking around in your My Site list until you restarted the app.
+* [***] My Site: your My Site screen now has two tabs, "Site Menu" and "Home". Under "Home", you'll find contextual cards with some highlights of whats going on with your site. Check your drafts or scheduled posts, your today's stats or go directly to another section of the app. [https://github.com/wordpress-mobile/WordPress-Android/issues/15989]
+* [*] Site creation: Adds a new screen asking the user the intent of the site [https://github.com/wordpress-mobile/WordPress-Android/pull/16306]
+* [*] Notifications: Fixed notification detail crash for comments with File block [https://github.com/wordpress-mobile/WordPress-Android/pull/16314]
+* [**] Block Editor: Quote block: Adds support for V2 behind a feature flag [https://github.com/WordPress/gutenberg/pull/40133]
```

Comment:
Hey @AliSoftware 👋 this shouldn't be included, I'm sorry it was included in the main apps release notes 😞 

Since this was merged already, what is the process to have it removed? Let me know how I can help with that.

Thanks for asking about this!

Reply:
Doing a PR to remove them is usually the way to go;

But since @mokagio already pinged our freelance writer in our #wordpress-release-notes in C02S5P5MBGA-p1650432777950529-slack I'd suggest you leave a message to her in that channel ASAP to let her know too that she should omit that item in the public-facing notes.

Reply:
Btw if that item was originally included in the `RELEASE-NOTES.txt` file so that it gets beta-tested and included in the CfT even if not yet public-facing because behind a feature flag, I recently learned that using `[internal]` for such cases is then usually the way to go (ie "[inernal]" meaning "include in CfT but not public facing notes".

If that's the case for this item here, be sure to let @thehenrybyrd know that she should include the item (or not) in her CfT

Reply:
Cool! Thanks! I already sent a message there.

Reply:
TIL about `[internal]` I'll have it in mind for future updates. In this case, we do want to test it because it will be enabled in a few weeks cc @thehenrybyrd

Reply:
@geriux as I understand this can be tested with a self-hosted site with `Quote Block v2` enabled in Gutenberg Experiment settings; I've included this in the internal CfT :)

Reply:
> as I understand this can be tested with a self-hosted site with Quote Block v2 enabled in Gutenberg Experiment settings; 

That's correct, awesome! Thank you!

Reply:
Thank you for including that detail in the PR notes!